### ISSUE TITLE
ConnectionProvider not found

### ISSUE THREAD

**Issue by Lokuc** [heroku/heroku.jar#126]
HerokuAPI h = new HerokuAPI("APIKEY");

Exception in thread "main" java.lang.IllegalArgumentException: ConnectionProvider not found
	at com.heroku.api.connection.ConnectionFactory.get(ConnectionFactory.java:21)
	at com.heroku.api.HerokuAPI.<init>(HerokuAPI.java:73)
	at com.severgames.telegram.Test.<init>(Test.java:19)
	at com.severgames.telegram.Test.main(Test.java:14)

**Comment by ErrorxCode** [comment]
The same problem, Have you resolved this issue?

**Comment by ErrorxCode** [comment]
## Solution
### Add anyone of the two :-
**Jackson**
```xml
<dependency>
    <groupId>com.heroku.api</groupId>
    <artifactId>heroku-json-jackson</artifactId>
    <version>0.25</version>
</dependency>
```
**Apache client**
```xml
<dependency>
    <groupId>com.heroku.api</groupId>
    <artifactId>heroku-http-apache</artifactId>
    <version>0.25</version>
</dependency>
```

**Comment by Malax** [comment]
@ErrorxCode is correct, but please note that there is a much more recent version available than `0.25`.
